vignettes/submodel_documentation_overview.Rmd

```diff
@@ -0,0 +1,394 @@
+
+- `run_year` year of run
+- `age` age of fish
+- `released` total number of fish released at hatchery
+- `flow_oct_nov` the median flow for October and November
+- `flow_apr_may` the median flow for April and May
+- `mean_pdo_return` environmental conditions 
+
+And based on the location of fish assigns a transport distance to bay, and then computes a stray rate for each release group. 
+
+*Source*
+[Sturrock et al. (2019)](https://afspubs.onlinelibrary.wiley.com/doi/epdf/10.1002/fsh.10267) Examining eight decades of hatchery release data in California’s Central Valley.
+
+Please see the [stray rate documentation](https://reorienting-to-recovery.s3.us-west-1.amazonaws.com/updated-stray-rates.html) for more information on how Sturrock et al. (2019) stray logic is incorporated into the R2R DSMs. 
+
+### Adult Prespawn Survival
+Adult prespawn survival is a function of migratory temperatures and whether or not the bypass is overtopped. This function also contains a calibrated intercept.  In the model, this information is contained in the following data objects: 
+
+- data: `degree_days`
+- data: `month_return_proportions`
+- coefficent: `.adult_prespawn_deg_day`
+- calibrated intercept: `..surv_adult_prespawn_int` 
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+*Source*
+
+- `.adult_prespawn_deg_day` coefficent from Colvin et al. (2018)
+- `month_return_proportions` TODO
+- *Calibrated intercept.*
+
+
+### Spawn Success & Egg to Fry Survival 
+
+
+Adult spawning success and egg to fry survival are a function of scouring, sex ratios, redd size, and fecundity, and temperature. In the model, this information is contained in the following data objects: 
+
+- data: `prob_nest_scoured`
+- data: `spawn_success_sex_ratio`
+- data: `spawn_success_redd_size`
+- data: `spawn_success_fecundity`
+- data: `fecundity_lookup`
+- data: `mean_egg_temp_effect`
+- calibrated intercept: `..surv_egg_to_fry_int`
+- coefficient: `.surv_egg_to_fry_proportion_natural`	
+- coefficient: `.surv_egg_to_fry_scour`
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+*Source*
+
+- `prob_nest_scoured`: Experts estimated probabilities for predation, stranding, and nest scouring
+- `spawn_success_sex_ratio`: Variable describing the female to male spawning ratio, default 0.5, source: expert opinion from SIT members.
+- `spawn_success_redd_size`: Variable describing the size of redds including defensible space, default value 9.29 square meters, source: expert opinion from SIT members.
+- `spawn_success_fecundity`: Variable describing the number of eggs per female, default value 5522, source: Moyle, P. B. 2002. Inland Fishes of California. University of California Press, Berkeley CA
+- Updated to have fecundity vary by size of fish, Roni and Quinn 1995 logic was used to map fu=ish age to size. Then we scaled fecundity by size using logic from Malick et all 2023. 
```

Comment:
typo and same comment re how to link papers - maybe we should discuss this strategy at a high level for the vignette as a whole!

Reply:
updated and made these links